bin/rbenv-doctor

```diff
@@ -49,7 +67,7 @@ fi
 warnings=0
 
 echo -n "Checking for \`rbenv' in PATH: "
-num_locations="$(command-list rbenv | uniq | count-lines)"
+num_locations="$(command-list rbenv | remove-bin-symlink | uniq | count-lines)"
```

Comment:
How about just piping `command-list` output to a simple `sed 's:^/usr/bin/:/bin/:'`? It will work fine for counting since there is a `uniq` later in the pipe as well.

Reply:
I like that 👌